releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java

```diff
@@ -70,7 +70,6 @@ public void testGenerateOnlyBreakingCompatibility() throws Exception {
 
         assertFile("xdocBreakingCompatibility.txt", MainProcess.XDOC_FILENAME);
         assertFile("twitterGplusBreakingCompatibility.txt", MainProcess.TWITTER_FILENAME);
```

Comment:
Will you remove google plus from file name?

Reply:
I tried to minimize changes .. 
Ok, I will remove

Reply:
done